lib/YaST/DNSServer/MainPage.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
```

Comment:
There is not 'Main' page, this page is 'StartUp' and it is just the one the user lands first.
Please rename all occurences.